dockstore-webservice/src/main/java/io/dockstore/webservice/resources/Description.java

```diff
@@ -40,7 +40,7 @@
 @Provider
 @SwaggerDefinition(info = @Info(description =
         "This describes the dockstore API, a webservice that manages pairs of Docker images and associated metadata such as "
-                + "CWL documents and Dockerfiles used to build those images", version = "1.3.0", title = "Dockstore API", contact = @Contact(name = "Dockstore@ga4gh", email = "theglobalalliance@genomicsandhealth.org", url = "https://github.com/ga4gh/dockstore"), license = @License(name = " GNU Lesser General Public License", url = "https://www.gnu.org/licenses/lgpl-3.0.en.html")), consumes = "application/json", produces = "application/json", tags = {
+                + "CWL documents and Dockerfiles used to build those images", version = "1.4.4", title = "Dockstore API", contact = @Contact(name = "Dockstore@ga4gh", email = "theglobalalliance@genomicsandhealth.org", url = "https://github.com/ga4gh/dockstore"), license = @License(name = "Apache License Version 2.0", url = "https://github.com/ga4gh/dockstore/blob/develop/LICENSE")), consumes = "application/json", produces = "application/json", tags = {
```

Comment:
possible to grab the version from pom?

Reply:
Haven't found a clean way of doing it due to https://stackoverflow.com/questions/16509065/get-rid-of-the-value-for-annotation-attribute-must-be-a-constant-expression-me?noredirect=1&lq=1